src/controllers/confirm.company.controller.ts

```diff
@@ -24,26 +24,27 @@ export const get = async (req: Request, res: Response, next: NextFunction) => {
 export const post = async (req: Request, res: Response, next: NextFunction) => {
   const session: Session = req.session as Session;
   const companyNumber = req.query["companyNumber"] as string;
+  const company: CompanyProfile = await getCompanyProfile(req.query.companyNumber as string);
   try {
     const eligibilityStatusCode: EligibilityStatusCode = await checkEligibility(session, companyNumber);
     if (eligibilityStatusCode === EligibilityStatusCode.COMPANY_VALID_FOR_SERVICE) {
       await createNewConfirmationStatement(req);
       const url = TRADING_STATUS_PATH.replace(`:${urlParams.PARAM_COMPANY_NUMBER}`, companyNumber);
       return res.redirect(url);
     } else {
-      return displayEligibilityStopPage(res, eligibilityStatusCode);
+      return displayEligibilityStopPage(res, eligibilityStatusCode, company);
     }
   } catch (e) {
     return next(e);
   }
 };
 
-const displayEligibilityStopPage = (res: Response, eligibilityStatusCode: EligibilityStatusCode) => {
+const displayEligibilityStopPage = (res: Response, eligibilityStatusCode: EligibilityStatusCode, company: CompanyProfile) => {
   const stopPage: string = stopPages[eligibilityStatusCode];
   if (!stopPage) {
     throw new Error(`Unknown eligibilityStatusCode ${eligibilityStatusCode}`);
   }
-  return res.render(stopPage);
+  return res.render(stopPage, { company, templateName: Templates.USE_PAPER });
```

Comment:
The templateName is specific to the USE_PAPER screen only, but so far we have 3 stop screens that will use this function and they'll all get the same templateName of USE_PAPER. The stopPage const will have the template name of the specific stop page so we can use that instead eg.
`return res.render(stopPage, { company, templateName: stopPage });`